scripts/run_integration_tests_aarch64.sh

```diff
@@ -223,7 +223,7 @@ update_workloads() {
         pushd $WORKLOADS_DIR
         git clone "https://gitlab.com/virtio-fs/virtiofsd-rs.git" $VIRTIOFSD_RS_DIR
         pushd $VIRTIOFSD_RS_DIR
-        git checkout 21d20035a582fb0389697b1bd7f8331623a77939
+        git checkout v1.1.0
         time cargo build --release
         cp target/release/virtiofsd-rs $VIRTIOFSD_RS || exit 1
```

Comment:
Looks like they chose to rename the binary from `virtiofsd-rs` to `virtiofsd`, which is why the CI failed..

Reply:
Also, don't forget to update the `fn prepare_virtiofsd_rs_daemon()` from "integration.rs" if we decide to follow this renaming.